service/RTC/MediaType.js

```diff
@@ -1,15 +1,9 @@
 /**
- * Enumeration of RTC media stream types
- * @type {{AUDIO: string, VIDEO: string}}
+ * The audio type.
  */
-var MediaType = {
-    /**
-     * The audio type.
-     */
-    AUDIO: "audio",
-    /**
-     * The video type.
-     */
-    VIDEO: "video"
-};
-module.exports = MediaType;
\ No newline at end of file
+export const AUDIO = "audio";
```

Comment:
ok, i think it's a bit of a bummer to lose some 'scoping' for the term and having everything essentially in the top-level namespace means we have to build the scoping into the name itself.  i think this is fine since you were consistent, i'll have to read more about the idiomatic way to do this in es6.